src/components/App/App.js

```diff
@@ -2,28 +2,38 @@ import React, { Component } from 'react';
 import movieData from '../../movieData';
 import Collection from '../Collection/Collection';
 import Nav from '../Nav/Nav';
+import MovieInfoView from '../MovieInfoView/MovieInfoView';
 
 import './App.css';
 
 class App extends Component {
   constructor() {
     super();
     this.state = {
-      movies:  movieData.movies
+      movies:  movieData.movies,
+      collectionView: true,
+      currentMovie: null
+    }
+  }
+
+  handleMovieView = (id) => {
+    if(id === "button"){
+      this.setState({collectionView: true})
+    } else if(id){
+      const movie = this.state.movies.find(movie => movie.id === +id)
+      this.setState({collectionView: false, currentMovie: movie})
```

Comment:
Nice use of a single setState() call to update more than one piece of state at once!